src/common/logging.cpp

```diff
@@ -49,9 +49,14 @@
 #include "common/code_utils.hpp"
 #include "common/time.hpp"
 
-static otbrLogLevel sLevel            = OTBR_LOG_INFO;
-static const char   sLevelString[][8] = {
-      "[EMERG]", "[ALERT]", "[CRIT]", "[ERR ]", "[WARN]", "[NOTE]", "[INFO]", "[DEBG]",
+static otbrLogLevel sLevel = OTBR_LOG_INFO;
+
+#if OTBR_ENABLE_DETAILED_LOGGING_FLAG
+static otbrLogLevel sDefaultLevel = OTBR_LOG_INFO;
+#endif
```

Comment:
Is it possible to move this part below `sLevelString` so that the diff won't involve format changes.

Reply:
Done